test/unit/server.js

```diff
@@ -371,24 +371,37 @@ describe('Server', function () {
     var queue = Object.keys(tasks)[0]
 
     beforeEach(function () {
-      sinon.spy(server, 'setTask')
+      sinon.stub(server, 'setTask')
+    })
+
+    afterEach(function () {
+      server.setTask.restore()
     })
 
     it('should set multiple tasks', function () {
       var numTasks = Object.keys(tasks).length
       server.setAllTasks(tasks)
-      assert.equal(server.setTask.callCount, numTasks)
-      assert.equal(Object.keys(server._tasks).length, numTasks)
+      sinon.assert.callCount(server.setTask, numTasks)
     })
 
     describe('with option objects', function () {
+      beforeEach(function () {
+        sinon.stub(server.log, 'warn')
+      })
+
+      afterEach(function () {
+        server.log.warn.restore()
+      })
+
       it('should throw if a task is not defined', function () {
         var tasks = {}
         tasks[queue] = { msTimeout: 2000 }
-        assert.throws(function () {
-          server.setAllTasks(tasks)
-        }, /No task function defined/)
-        assert.equal(server.setTask.callCount, 0)
+        server.setAllTasks(tasks)
+        sinon.assert.calledOnce(server.log.warn)
+        sinon.assert.calledWith(server.log.warn,
+          sinon.match.object, 'ponos.Server.setAllTasks: No task function defined')
```

Comment:
Put each of these arguments on new lines.

```js
sinon.assert.calledWith(
  server.log.warn,
  sinon.match.object,
  'ponos.Server.setAllTasks: No task function defined'
)
```

Generally, for me, if a line break into a list of arguments, all of them go onto their own line.